packages/create-analog/template-angular-v15/src/main.ts

```diff
@@ -0,0 +1,14 @@
+import './polyfills';
+import { enableProdMode } from '@angular/core';
+import { bootstrapApplication } from '@angular/platform-browser';
+import { provideFileRouter } from '@analogjs/router';
+
+import { AppComponent } from './app/app.component';
+
+if (import.meta.env.PROD) {
+  enableProdMode();
+}
```

Comment:
Delete these lines and imports as its no longer needed

```suggestion

```

Reply:
Nice catch :D